sopel/coretasks.py

```diff
@@ -1009,4 +1009,4 @@ def handle_url_callbacks(bot, trigger):
         for function, match in bot.search_url_callbacks(url):
             # trigger callback defined by the `@url` decorator
             if hasattr(function, 'url_regex'):
-                function(bot, trigger, match=match)
+                bot.call(function, bot, trigger, match=match)
```

Comment:
You can remove the `match` argument by doing this instead:

```
@wraps(function)
def decorated(bot, trigger):
    return function(bot, trigger, match=match)
```

And voilà. Nothing to do in `bot.call` now.